src/chrome/content/rules/Sparkasse-ansbach.de.xml

```diff
@@ -1,5 +1,15 @@
+<!--
+
+Connection closed:
+
+sparkasse-ansbach.de	
+Fetch error: http://sparkasse-ansbach.de/ => https://sparkasse-ansbach.de/: (7, 'Failed to connect to sparkasse-ansbach.de port 443: Connection refused')
```

Comment:
Please just write something less wordy like:
```
Secure connection refused:
	- sparkasse-ansbach.de
```

You can find various examples in existing rules.